src/lib/actions.js

```diff
@@ -217,3 +217,15 @@ export const setLanguage = store.action((state, preferredLanguage) => {
     userPreferredLanguage: preferredLanguage,
   };
 });
+
+export const closeToC = store.action(() => {
+  return {
+    tocOpened: false,
```

Comment:
change to isTocOpened. That'll match other booleans like isModalOpen, isSideNavExapnded, etc